edx-platform/wikilearn/lms/templates/footer.html

```diff
@@ -23,16 +23,7 @@
       <nav class="nav-colophon" aria-label="${_('About')}">
         <ol>
           <li class="nav-colophon">
-            <a href="#">Privacy Policy</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">About Wikimedia</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">Disclaimers</a>
-          </li>
-          <li class="nav-colophon">
-            <a href="#">Cookie Statement</a>
+            <a href="https://foundation.wikimedia.org/wiki/Privacy_policy" target="_blank">Privacy Policy</a>
```

Comment:
```suggestion
            <a href="https://foundation.wikimedia.org/wiki/Privacy_policy" target="_blank">${_("Privacy Policy")}</a>
```

mark Privacy Policy as a translation string